.github/workflows/process_opened_issues.yml

```
@@ -11,7 +11,7 @@ jobs:
     name: Adding issue to the IREE GitHub project
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/add-to-project@b56023be57e9c76f06e3844f18a1ab5dec7e7617
+      - uses: actions/add-to-project@7a0820f97673dfefc999713a9a6d6b7ee128bba5
```

document the intended version in a comment, as with our other actions (see https://github.com/google/iree/pull/8997 for examples)

```suggestion
      - uses: actions/add-to-project@7a0820f97673dfefc999713a9a6d6b7ee128bba5  # v0.0.3
```